sample/benchmark/src/main/java/com/datadog/benchmark/sample/BenchmarkApplication.kt

```diff
@@ -0,0 +1,128 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.benchmark.sample
+
+import android.annotation.SuppressLint
+import android.app.Application
+import android.util.Log
+import com.datadog.android.Datadog
+import com.datadog.android.core.configuration.BackPressureMitigation
+import com.datadog.android.core.configuration.BackPressureStrategy
+import com.datadog.android.core.configuration.BatchSize
+import com.datadog.android.core.configuration.Configuration
+import com.datadog.android.core.configuration.UploadFrequency
+import com.datadog.android.privacy.TrackingConsent
+import com.datadog.android.rum.Rum
+import com.datadog.android.rum.RumConfiguration
+import com.datadog.android.rum.tracking.NavigationViewTrackingStrategy
+import com.datadog.benchmark.DatadogExporterConfiguration
+import com.datadog.benchmark.DatadogMeter
+import com.datadog.sample.benchmark.BuildConfig
+import com.datadog.sample.benchmark.R
+
+internal class BenchmarkApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        initDatadogSessionReplay()
+        enableDatadogMeter()
+    }
+
+    private fun enableDatadogMeter() {
+        DatadogMeter.create(
+            DatadogExporterConfiguration.Builder(BuildConfig.BM_API_KEY)
+                .setApplicationId(BuildConfig.APPLICATION_ID)
+                .setApplicationName("Benchmark Application")
+                .setApplicationVersion(BuildConfig.VERSION_NAME)
+                .setIntervalInSeconds(METER_INTERVAL_IN_SECONDS)
+                .build()
+        ).startGauges()
+    }
+
+    private fun initDatadogSessionReplay() {
+        Datadog.initialize(
+            this,
+            createDatadogConfiguration(),
+            TrackingConsent.GRANTED
+        )
+        val rumConfig = createRumConfiguration()
+        Rum.enable(rumConfig)
+    }
+
+    private fun createRumConfiguration(): RumConfiguration {
+        return RumConfiguration.Builder(BuildConfig.BM_RUM_APPLICATION_ID)
+            .useViewTrackingStrategy(
+                NavigationViewTrackingStrategy(
+                    R.id.nav_host_fragment,
+                    true,
+                    BenchmarkNavigationPredicate()
+                )
+            )
+            .setTelemetrySampleRate(SAMPLE_RATE_TELEMETRY)
+            .trackUserInteractions()
+            .trackLongTasks(THRESHOLD_LONG_TASK_INTERVAL)
+            .trackNonFatalAnrs(true)
+            .setViewEventMapper { event ->
+                event.context?.additionalProperties?.put(ATTR_IS_MAPPED, true)
```

Comment:
what's the reason for these mappers ?

Reply:
This is a copy-paste from sample application, I think for the benchmark we should cover all the usage, does it make sense to you?